core/db/db_works.py

```diff
@@ -55,13 +57,18 @@ def set_status(self, status: StatusChoices) -> bool:
 
     def add_peer(self,
                  public_key: str,
+                 private_key: str,
                  preshared_key: str,
                  shared_ips: str,
                  peer_name: str = None) -> ConnectionPeer:
+        private_peer_key = generate_private_key()
+        public_peer_key = generate_public_key(private_peer_key)
+        preshared_peer_key = generate_preshared_key()
```

Comment:
consider making these arguments optional. generate pub_key, presh_key and private_key if they're aren't specified, use given otherwise

Reply:
This is the main functionality of the bot - to generate tokens and help users to connect. Also it isn't comfortable to put more than 100 symbols as keys via bot